app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetControllerV2.kt

```diff
@@ -47,14 +47,15 @@ class BloodPressureEntrySheetControllerV2 @Inject constructor(
     val replayedEvents = ReplayUntilScreenIsDestroyed(events)
         .compose(ReportAnalyticsEvents())
         .compose(combineDateInputs())
+        .compose(validateBpInput())
```

Comment:
I'm really happy we took the time to standardize this now.

Reply:
I also think we should rename `UiEvent` -> `Event` at this point.

Reply:
Agreed